internal/resources/testing/test_helper.go

```diff
@@ -135,14 +133,6 @@ func WithEKSCluster() TestAcceptanceOption {
 				config.KubernetesVersion = val
 			}
 
-			if val, exists := os.LookupEnv("EKS_LAUNCH_TEMPLATE_NAME"); exists {
```

Comment:
we don't required this env var to check ?

Reply:
No, we don't. Made the config changes accordingly